governance/program/tests/process_remove_required_signatory.rs

```diff
@@ -0,0 +1,231 @@
+#![cfg(feature = "test-sbf")]
+
+mod program_test;
+
+use solana_program_test::tokio;
+
+use program_test::*;
+use solana_program::pubkey::Pubkey;
+
+use crate::cookies::{GovernanceCookie, RealmCookie, TokenOwnerRecordCookie};
+use solana_sdk::signature::{Keypair, Signer};
+use spl_governance_tools::error::GovernanceToolsError;
+
+async fn set_up_governance_with_required_signatory(
```

Comment:
nit: This is very cool !!!. Can we move it to `GovernanceProgramTest` and rename to `with_governance_with_required_signatory`? The `with_` prefix indicates case/scenario builder methods

Reply:
resolved